protos/repository.proto

```diff
@@ -19,7 +19,7 @@ message BentoUri {
 
   StorageType type = 1;
   string uri = 2;
-  string s3_presigned_url = 3;
```

Comment:
we should avoid this in the future, especially given more users are using remote YataiService and there will be backward compatibility issues